test/unit/foreman/access_permissions_test.rb

```diff
@@ -1,6 +1,7 @@
 require 'test_helper'
 require 'foreman/access_control'
```

Comment:
These foreman/access_* requires should be in the shared file, making it more reusable (as that's where they're referenced).